assets/app/actions/makeCommitData.js

```diff
@@ -0,0 +1,29 @@
+import { encodeB64 } from '../util/encoding';
```

Comment:
Putting it into a private function makes the tests much messier as it increases the paths we need to test. It's not an "action", that's for sure (see earlier, concurring comment).